README.md

```diff
@@ -273,28 +273,28 @@ The following options are supported by all filters except `Callback` and `Finder
 - `after` (`bool`, defaults to `false`) Whether to automatically add a wildcard
   *after* the search term.
 
-- ~~`mode`~~ (`string`, default to `or`) **This options is deprecated**, please use
+- ~~`mode`~~ (`string`, default to `OR`) **This options is deprecated**, please use
```

Comment:
@ADmad You can take the superfluous **s** from **options** :)

https://www.youtube.com/watch?v=KzyCi1BFATA